include/GL/glui.h

```diff
@@ -442,7 +444,7 @@ class GLUIAPI GLUI_StdBitmaps
 {
 public:
     GLUI_StdBitmaps(); 
-    ~GLUI_StdBitmaps();
+    ~GLUI_StdBitmaps() = default;
 
```

Comment:
The destructor can be omitted. Using = default leaves no workarounds for C++98 compilers. It locks into C++11 for no real benefit. 2011 was only 6 years ago.